system_test/aest_sync_SUITE.erl

```diff
@@ -31,7 +34,9 @@
 
 %=== MACROS ====================================================================
 
--define(MINING_TIMEOUT, 2000).
+-define(MINING_TIMEOUT,   2 * 1000).
+-define(STARTUP_TIMEOUT, 10 * 1000).
+-define(SYNC_TIMEOUT,     1 * 1000).
```

Comment:
Is this per block? Then it is very large, 50ms per block or something sounds reasonable?!